src/backend/libpq/rangerrest.c

```diff
@@ -366,7 +365,7 @@ static size_t write_callback(char *contents, size_t size, size_t nitems,
 	if (curl->response.buffer == NULL)
 	{
 		/* out of memory! */
-		elog(WARNING, "not enough memory for Ranger response");
+		elog(WARNING, "cannot allocate memory for ranger response");
```

Comment:
hmm, IMO , not enough memory is not the only error that will occur. There maybe other errors  in the repalloc implementation. So I changed the messages.